src/pkg/podman/podman.go

```diff
@@ -332,3 +333,83 @@ func SystemMigrate(ociRuntimeRequired string) error {
 
 	return nil
 }
+
+// internalError serves for representing errors printed by Podman to stderr
+type internalError struct {
+	errors []string
+}
+
+func (e *internalError) Error() string {
+	if e.errors == nil || len(e.errors) == 0 {
+		return ""
+	}
+
+	var builder strings.Builder
+	for i, part := range e.errors {
+		if i != 0 {
+			builder.WriteString(": ")
+		}
+		builder.WriteString(part)
+	}
+	return builder.String()
+}
+
+// Is lexically compares errors
+//
+// The comparison is done for every part in the error chain not across.
+func (e *internalError) Is(target error) bool {
+	if target == nil {
+		return false
+	}
+
+	if e.errors == nil || len(e.errors) == 0 {
+		return false
+	}
+
+	for {
+		if e.errors[0] == target.Error() {
+			return true
+		}
+
+		if e = e.Unwrap(); e == nil {
+			return false
+		}
+	}
+}
+
+func (e internalError) Unwrap() *internalError {
```

Comment:
Is this the [conventional](https://blog.golang.org/go1.13-errors) signature for `Unwrap`? Isn't it:
```
func (e *internalError) Unwrap() error {
    ...
}
```

Honest question. I could be wrong.

Reply:
It is not, the method should return `error`. Nice catch!